docs/pages/releases/2021-12-01.mdx

```diff
@@ -0,0 +1,101 @@
+import { Markdown, getStaticProps } from '../../components/Markdown';
+import { Emoji } from '../../components/primitives/Emoji';
+import { Stack } from '../../components/primitives/Stack';
+
+# Release: 1st December 2021
+
+This release marks the achievement of **General Availability** status for Keystone 6! <Emoji symbol="🚀" alt="Rocket"/>
+
+We’ve also included a range of improvements to Keystone’s TypeScript DX since shipping [last week’s release candidate](/releases/2021-11-24).
+
+## Keystone 6 <Emoji symbol="⚡️" alt="Lightning" />
+
+With this major release, the project has moved to the `@keystone-6` namespace on npm, and our version numbers have been reset.
+
+We **highly recommend you upgrade** your existing **Keystone Next** projects to **Keystone 6** with the packages below:
+
+```json
+"@keystone-6/keystone": "1.0.0",
```

Comment:
```suggestion
"@keystone-6/core": "1.0.0",
```

We need to note that `@keystone-next/keystone` has renamed to `@keystone-6/core`